src/Features/LanguageServer/Protocol/Handler/CodeActions/CodeActionHelpers.cs

```diff
@@ -141,13 +142,17 @@ private static bool IsCodeActionNotSupportedByLSP(IUnifiedSuggestedAction sugges
             LSP.CodeActionKind codeActionKind,
             LSP.Diagnostic[]? diagnosticsForFix,
             IUnifiedSuggestedAction suggestedAction,
-            ArrayBuilder<string> codeActionPathList,
+            ImmutableArray<string> codeActionPathList,
             bool isTopLevelCodeAction = false)
         {
             var codeAction = suggestedAction.OriginalCodeAction;
             using var _1 = ArrayBuilder<LSP.CodeAction>.GetInstance(out var nestedCodeActions);
 
-            codeActionPathList.Add(codeAction.Title);
+            if (!isTopLevelCodeAction)
```

Comment:
why not just always add the title?  i don't really get why we're special casing anything.